doc/source/examples/analysis/alignment_and_rms/rmsd.ipynb

```diff
@@ -73,6 +76,29 @@
     "ref = mda.Universe(PSF, CRD)  # open AdK (PDB ID: 4AKE)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background\n",
+    "\n",
+    "The root mean square deviation (RMSD) of particle coordinates is one measure of distance, or dissimilarity, between molecular conformations. Each structure should have matching elementwise atoms $i$ in the same order, as the distance between them is calculated and summed for the final result. It is calculated between coordinate arrays $\\mathbf{a}$ and $\\mathbf{b}$ according to the below equation:\n",
+    "\n",
+    "$$ RMSD(\\mathbf{a}, \\mathbf{b}) = \\sqrt{\\frac{1}{n} \\sum_{i=1}^{n}{|\\mathbf{a}_i-\\mathbf{b}_i|^2}} $$\n",
+    "\n",
+    "The RMSD between one reference structure and a trajectory of structures is often calculated as a way to measure the dissimilarity of the trajectory conformational ensemble to the reference. This reference is frequently the first frame of the trajectory (the default in MDAnalysis), in which case it can provide insight into the overall movement from the initial starting point. While stable RMSD values from a reference structure are frequently used as a measure of conformational convergence, this metric suffers from the problem of *degeneracy*: many different structures can have the same RMSD from the same reference. For an alternative measure, you could use [pairwise or 2D RMSD](pairwise_rmsd.ipynb).\n",
+    "\n",
+    "Typically not all coordinates in a structures are included in an RMSD analysis. With proteins, the fluctuation of the residue side-chains is not representative of overall conformational change. Therefore when RMSD analyses are performed to investigate large-scale movements on proteins, the atoms are usually restricted only to the backbone atoms (forming the amide-bond chain) or the alpha-carbon atoms. \n",
+    "\n",
+    "MDAnalysis provides functions and classes to calculate the [RMSD between coordinate arrays](#RMSD-between-two-sets-of-coordinates), and [Universes](#RMSD-of-a-Universe-with-multiple-selections) or [AtomGroups](#RMSD-of-an-AtomGroup-with-multiple-selections). \n",
+    "\n",
+    "The contribution of each particle $i$ to the final RMSD value can also [be weighted](#Weighted-RMSD-of-a-trajectory) by $w_i$:\n",
+    "\n",
+    "$$ RMSD(\\mathbf{a}, \\mathbf{b}) = \\sqrt{\\frac{\\sum_{i=1}^{n}{w_i|\\mathbf{a}_i-\\mathbf{b}_i|^2}}{\\sum_{i-1}^n w_i}} $$\n",
+    "\n",
+    "RMSD analyses are frequently weighted by mass. The MDAnalysis `RMSD` class allows you to both [select mass-weighting](#Mass) with `weights='mass'` or `weights_groupselections='weights'`, or to [pass custom arrays](#Custom-weights) into either keyword."
```

Comment:
Shouldn't this be `weights_groupselections='mass'`?

Going by: https://github.com/MDAnalysis/mdanalysis/blob/343e10a257690e4312673f2a0fa4703b31b2e223/package/MDAnalysis/analysis/rms.py#L382-L386

Reply:
Yes. Oop